tao_compiler/mlir/disc/transforms/fusion_utils.h

```diff
@@ -255,6 +252,23 @@ class FusionPatternBase {
 };
 
 // Represents a list of lmhlo ops that are going to be fused.
+// Concepts for a fusion pattern:
```

Comment:
nit: it would be better to add this part of doc for `GpuFusionPattern` instead of `FusionPattern` since stitch-cpu does not use these strategy.

Reply:
TBD in next pr.